addons/account_payment/wizards/res_config_settings.py

```diff
@@ -1,9 +1,77 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
     pay_invoices_online = fields.Boolean(config_parameter='account_payment.enable_portal_payment')
+
+    providers_state = fields.Selection(
+        selection=[
+            ('none', 'None'),
+            ('primary_provider', 'Primary Provider'),
+            ('other', 'Other Provider')
+        ],
+        compute='_compute_providers_state',
+    )
```

Comment:
Do we need 'none' as a separate selection option?, Instead for all the cases where this field should be none we can set it to false since it's not required.
```suggestion
    providers_state = fields.Selection(
        selection=[
            ('primary_provider', "Primary Provider"), ('other', "Other Provider")
        ],
        compute='_compute_providers_state',
    )
```